node-util/sbin/oo-admin-cartridge

```diff
@@ -124,7 +124,7 @@ parser  = OptionParser.new do |opts|
   opts.on('-o', '--cartridge_vendor VENDOR', 'Cartridge vendor of cartridge to erase. default: redhat') { |o| options.cartridge_vendor = o }
   opts.on('-n', '--name NAME', 'Name of cartridge to erase') { |o| options.name = o }
   opts.on('-v', '--version VERSION', 'Version of packaged software to erase') { |o| options.version = o }
-  opts.on('-c', '--cartridge_version VERSION', 'Version of cartridge to erase') { |o| options.cartridge_version = o }
+  opts.on('-c', '--cartridge_version VERSION', 'Version of cartridge to erase (Note: All software versions of given cartridge version will be erased.)') { |o| options.cartridge_version = o }
```

Comment:
I can modify the info to be on the 'v' option but the erase method requires both -c and -v option in order to retrieve the correct cartridge. I would rather not to mess with it as it's working fine. Also, requiring to have both version and cart version may help to avoid some weird corner cases when somehow they have same different versions with the same cartridge version.